Tests/js/apps/assets/ez-platformuiapp-tests.js

```diff
@@ -1477,19 +1515,52 @@ YUI.add('ez-platformuiapp-tests', function (Y) {
                 method: 'set',
                 args: ['id', undefined]
             });
+        },
+
+        "Should reset the user and provide the response when logging out": function () {
+            this._mockExpectations();
 
             this.app.logOut(function (err, resp) {
                 Y.Assert.areSame(
-                    logOutError, err, "The error of the logout should be provided"
+                    this.logOutError, err, "The error of the logout should be provided"
```

Comment:
connected to https://github.com/ezsystems/PlatformUIBundle/pull/474/files#r47475426, here `this` is actually the App so here as well, `this.logOutError` and `this.logOutResponse` are also evaluated to `undefined` and that's why the test are passing (and you are lucky).